packages/millicast-sdk/src/View.js

```diff
@@ -323,4 +400,47 @@ export default class View extends BaseWebRTC {
       })
     }
   }
-};
+
+  /**
+   * @typedef {Object} DRMMediaOptions
+   * @property {String} codec - h264 for video
+   * @property {String} encryption - cbsc or clear
+   * @property {String} [keyId] - decryption keyId when encryption is not clear
+   * @property {String} [iv] - fairplay iv when encryption is not clear
+   */
+  /**
+   * @typedef {Object} CastLabDRMOptions extends DRMOptions
```

Comment:
I'd suggest not using the Castlabs name here, since we might have an in-house implementation of DRM down the line

Reply:
But this is for castLab DRM options, we could name our in-house implementation DolbyDRMOptions. If don't use this, what name do you suggest?

Reply:
how do you feel about `vendorDRMOptions`

Reply:
What if there is another vendor?

Reply:
It's generic enough to still be true

Reply:
Resolving it - no legal implications of this, so not worth a long discussion :)